src/assets/css/markdownStyle.css

```diff
@@ -0,0 +1,87 @@
+h1 {
```

Comment:
I think we should add a parent class in the selector, otherwise all others elements will be affected